package.json

```diff
@@ -190,6 +191,7 @@
     "react-json-view-lite": "^0.9.5",
     "react-router": "^5.2.1",
     "react-router-dom": "^5.3.0",
+    "react-syntax-highlighter": "^15.5.0",
```

Comment:
needs to be exact version. I know some others here aren't, but it should be.

Reply:
fixed